.github/workflows/unit_tests.yml

```diff
@@ -119,18 +119,64 @@ jobs:
           unset PYTHONPATH
 
           # install easyconfigs via distribution package
-          python setup.py sdist
+          python setup.py sdist > /dev/null
           ls dist
-          pip install dist/easybuild-easyconfigs*tar.gz
+          pip install dist/easybuild-easyconfigs*tar.gz > /dev/null
 
           # robot-paths value should not be empty, but have an entry that includes easybuild/easyconfigs subdir
+          echo "eb --show-config"
           eb --show-config | tee eb_show_config.out
           grep "^robot-paths .*/easybuild/easyconfigs" eb_show_config.out
+
           # check whether some specific easyconfig files are found
+          echo "eb --search 'TensorFlow-1.14.*.eb'"
           eb --search 'TensorFlow-1.14.*.eb' | tee eb_search_TF.out
           grep '/TensorFlow-1.14.0-foss-2019a-Python-3.7.2.eb$' eb_search_TF.out
-          eb --search '^foss-2018b.eb' | tee eb_search_foss.out
-          grep '/foss-2018b.eb$' eb_search_foss.out
+
+          echo "eb --search '^foss-2019b.eb'"
+          eb --search '^foss-2019b.eb' | tee eb_search_foss.out
+          grep '/foss-2019b.eb$' eb_search_foss.out
+
+          # make sure CVS easyconfigs are included in installation (cfr. issue #10325)
+          echo "Searching for CVS easyconfigs..."
+          eb --search '^CVS-' | grep '/CVS-'
 
           # try installing M4 with system toolchain (requires ConfigureMake easyblock + easyconfig)
           eb --prefix /tmp/$USER/$GITHUB_SHA M4-1.4.18.eb
+
+  test-sdist:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python: [2.7, 3.6, 3.7]
```

Comment:
@boegel With this added you should have added those lines too: https://github.com/easybuilders/easybuild-easyconfigs/pull/10326/files#diff-78a178eea90bd7e691df50ce4c38bd56L27-L29

I guess you are now testing only 1 python?

Reply:
--> https://github.com/easybuilders/easybuild-easyconfigs/pull/10388

Reply:
Woops, thanks for the fix!